CMakeLists.txt

```diff
@@ -273,6 +271,9 @@ else()
     target_compile_definitions(testxgboost PUBLIC ${xgboost_defs})
     target_include_directories(testxgboost PUBLIC src/tree)    
     target_link_libraries(testxgboost GTest::main xgboost)
+    if(OpenMP_CXX_FOUND OR OPENMP_FOUND)    
+      set_property(TARGET testxgboost APPEND PROPERTY LINK_FLAGS ${OpenMP_CXX_FLAGS})
```

Comment:
I think `target_link_libraries(... PUBLIC ${OpenMP_CXX_FLAGS})` will work as well.

Reply:
That's better, thanks.